scripts/bash_lib.sh

```diff
@@ -642,28 +642,36 @@ collect_dependencies() {
     else
       suffix="main"
     fi
+
     echo "-----------------"  >> "./reqs-${suffix}.${old_or_new}"
     echo "$p:" >> "./reqs-${suffix}.${old_or_new}"
     if [ "$pkgtype" == "rpm" ] ; then
       rpm -q -R "$p" | awk '{print $1}' | sort | grep -vE '/usr/bin/env|/usr/bin/bash' >> "./reqs-${suffix}.${old_or_new}"
-      filelist=$(rpm -ql "$p" | sort)
     else
       # We need sudo here for the apt-cache command, not for redirection
       # shellcheck disable=SC2024
-      sudo apt-cache depends "$p" --no-suggests --no-conflicts --no-breaks --no-replaces --no-enhances | sort >> "./reqs-${suffix}.${old_or_new}"
-      filelist=$(dpkg-query -L "$p" | sort)
+      sudo apt-cache depends "$p" --no-suggests --no-conflicts --no-breaks --no-replaces --no-enhances | sort >> "./reqs-${suffix}.${old_or_new}"  
     fi
-    echo "====== Package $p" >> "./ldd-${suffix}.${old_or_new}"
-    for f in $filelist ; do
-      # We do want to match literally here, not as regex
-      # shellcheck disable=SC2076
-      if [[ "$f" =~ "/.build-id/" ]] ; then
-        continue
+
+    # Collect LDD output for files installed by the package on the system
+    if [[ ${p} != "mariadb-test"* ]]; then
```

Comment:
In RPMs the package is named MariaDB-test. Unless I'm mistaken, the check above is case-sensitive, so it won't work.

Reply:
Made pkg name lowercase for this comparison
https://github.com/MariaDB/buildbot/pull/584/commits/6875da772f897ac0ed5ba36276bb9c7ba1694db1